thumb.te

```diff
@@ -143,6 +143,8 @@ optional_policy(`
     dbus_connect_session_bus(thumb_t)
 	dbus_stream_connect_session_bus(thumb_t)
 	dbus_chat_session_bus(thumb_t)
+	dbus_stream_connect_system_dbusd(thumb_t)
```

Comment:
Why is this rule needed when you also adding macro "dbus_system_bus_client(thumb_t)" ?

Reply:
I was running off other policies containing both the macros so I haven't realize the second one already contains stream_connect_pattern().
Changed.